web/peer_challenge_views.py

```
@@ -316,7 +316,7 @@ def calculate_leaderboard(challenge):
 
     # Now get all completed invitations
     invitations = PeerChallengeInvitation.objects.filter(
-        challenge=challenge, status="completed", user_quiz__isnull=False
+        challenge=challenge, status="completed", user_quiz__isnull=False, participant__profile__is_profile_public=True
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)